localisation/00_zuwc_united_localisation.csv

```diff
@@ -3004,6 +3004,57 @@ String_why_Azeroth_was_so_much_better;why Azeroth was so much better a few years
 String_Elune_and_deer;how could Elune give birth to a baby from a deer;;;;;;;;;;;;;x
 String_items_depend_on_who_enters_first;whether the items in a room are generated when someone enters;;;;;;;;;;;;;x
 nick_the_vile;the Vile;;[From.Getderdie] �ble;;;;;;;;;;;x
+beast_panther;Tamed Prowler;;Gez�hmter Pirscher;;;;;;;;;;;x
+beast_panther_desc;This character has tamed a prowler.;;Dieser Charakter hat einen Pirscher gez�hmt.;;;;;;;;;;;x
+beast_tortoise;Tamed Tortoise;;Gez�hmte Landschildkr�te;;;;;;;;;;;x
+beast_tortoise_desc;This character has tamed a tortoise.;;Dieser Charakter hat eine Landschildkr�te gez�hmt.;;;;;;;;;;;x
+beast_snake;Tamed Cobra;;;;;;;;;;;;;x
+beast_snake_desc;This character has tamed a cobra.;;;;;;;;;;;;;x
+beast_bear;Tamed Bear;;Gez�hmter B�r;;;;;;;;;;;x
+beast_bear_desc;This character has tamed a bear.;;Dieser Charakter hat einen B�ren gez�hmt.;;;;;;;;;;;x
+text_effect_tame_beast;Enables �GTame Beast�! on your portrait;;;;;;;;;;;;;x
+EVTDESC_WCCLS_110;The time has come for you to finally tame your own beast, as every great hunter should have a feral companion. However, you should be aware of the potential consequences, as taming a wild animal can leave a hunter wounded, or even killed. Now you only have to decide what kind of beast you want to tame.;;Es ist an der Zeit, dein eigenes Wildtier zu z�hmen, da jeder gro�e J�ger seinen eigenen wilden Gef�hrten haben sollte. Jedoch sollten dir die m�glichen Konsequenzen bewusst sein, da das Z�hmen eines Biests den J�ger verletzen oder sogar t�ten kann. Jetzt musst du dich nur noch entscheiden, was f�r eine Art von Tier du z�hmen m�chtest.;;;;;;;;;;;x
+EVTOPTA_WCCLS_110;A stalwart tortoise.;;Eine unersch�tterliche Landschildkr�te.;;;;;;;;;;;x
+EVTOPTB_WCCLS_110;A charming cobra.;;;;;;;;;;;;;x
+EVTOPTC_WCCLS_110;A stealthy panther.;;;;;;;;;;;;;x
+EVTOPTD_WCCLS_110;A ferocious bear.;;Ein angriffslustiger B�r.;;;;;;;;;;;x
+tame_beast;Tame Beast;;Wildtier z�hmen;;;;;;;;;;;x
+tame_beast_desc;Tame a wild beast and gain a companion.;;Z�hme ein wildes Tier als Gef�hrten.;;;;;;;;;;;x
+EVTDESC_HFP_11025_beast_panther;My personal beastly assassin creates chaos among enemies [battlefield_location.GetInOn] the [battlefield_location.GetTerrain], disrupting their formations. However, I did not see that stealthy panther for quite a lot time already...;;;;;;;;;;;;;x
+EVTDESC_HFP_11025_beast_tortoise;A extra thick shell and a deadly beak can be a very useful combination in battle if used wisely - and, not less importantly, without hurry. However, sometimes that downside can be pretty nerve-wracking, for example... right now.;;;;;;;;;;;;;x
+EVTDESC_HFP_11025_beast_bear;Not many people can stand still instead of running in fear after meeting incarnation of pure strength - my bear, even fewer can defeat it. But since there is no daring ones, it can't end badly, isn't it?..;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_HFP_11025_beast_bear;Not many people can stand still instead of running in fear after meeting an incarnation of pure strength - my bear, and even fewer can defeat it. But since no one is willing to attempt such a feat, it can only end well, right?..;;;;;;;;;;;;;x
```

Reply:
changed